src/DNA/native/src/PInvoke.c

```diff
@@ -95,7 +95,7 @@ fnPInvoke PInvoke_GetFunction(tMetaData *pMetaData, tMD_ImplMap *pImplMap) {
 
 	libName = MetaData_GetModuleRefName(pMetaData, pImplMap->importScope);
 
-#ifndef _WIN32
+#ifndef NO_JS
```

Comment:
@eloraiby Would you consider renaming this to NO_JS_INTEROP? It's also fine if you prefer to keep it the way it is.
I previously had the same define here named NO_JS_INTEROP but took it out to merge some upstream changes.